sg-bastion.tf

```diff
@@ -6,12 +20,12 @@ resource "alicloud_security_group" "bastion-sg" {
 }
 
 resource "alicloud_security_group_rule" "bastion-ssh-in" {
```

Comment:
This resource should probably have a more generic name now that it's being used in a count

Reply:
Moved rules from bastion to node sg and adjusted. Closed.